autopush-common/Cargo.toml

```diff
@@ -24,7 +24,7 @@ futures = "0.1.29" # 0.1.30 requires hyper 0.13+
 futures-backoff = "0.1.0"
 hex = "0.4.2"
 httparse = "1.3"
-hyper = "^0.12.36" # 0.13+ requires too many changes
+hyper = "0.14"
```

Comment:
sadly we still need this pinned to 0.12

Reply:
Ah, I was hoping that since a bunch of the 0.13 changes had been regressed we could skip the version. Oh well, i'll update the comment too.